src/status_im/translations/ne.cljs

```diff
@@ -0,0 +1,314 @@
+(ns status-im.translations.ne)
+
+(def translations
+  {
+   ;;common
+   :members-title                         "सदस्यहरु"
+   :not-implemented                       "कार्यान्वयन भएको छैन"
+   :chat-name                             "च्याट नाम"
+   :notifications-title                   "सूचना र ध्वनि"
+   :offline                               "अफ्लाइन"
+   :search-for                            "॰॰॰ को लागि खोज"
+   :cancel                                "रद्द गर्नुहोस्"
+   :next                                  "अर्को"
+   :type-a-message                        "सन्देश टाइप गरौं ॰॰॰"
+   :type-a-command                        "आदेश टाइप गरौं ॰॰॰"
+   :error                                 "त्रुटि"
+
+   :camera-access-error                   "कृपया क्यामेरा चलाउन आवश्यक अनुमति प्रदान गर्न, सिस्टम सेटिङ्ग भित्र गएर स्टेटस > क्यमेरा चयन भएको स्थिति निश्चित गर्नुहोस् ।"
+   :photos-access-error                   "कृपया फोटो चलाउन आवश्यक अनुमति प्रदान गर्न, सिस्टम सेटिङ्ग भित्र गएर स्टेटस > फोटो चयन भएको स्थिति निश्चित गर्नुहोस् ।"
+
+   ;;drawer
+   :invite-friends                        "साथीहरुलाई निम्त्याउं"
+   :faq                                   "प्राय सोधिने प्रश्नोत्तर"
+   :switch-users                          "उपयोगकर्ता बदलौं"
+   :feedback                              "प्रतिक्रिया"
+   :view-all                              "सबै हेर्ने"
+   :current-network                       "चल्ती नेटवर्क"
+
+   ;;chat
+   :is-typing                             "ले टाइप गर्दै हुनुहुन्छ"
+   :and-you                               "अनि तपाईं"
+   :search-chat                           "च्याटमा खोजौं"
+   :members                               {:one "१ सदस्य"}
+   :other                                 "{{count}} सदस्यहरु"
+   :zero                                  "सदस्यहीन"
+   :members-active                        {:one   "१ सदस्य"
+                                           :other "{{count}} सदस्यहरु"}
+   :zero                                  "सदस्यहीन"
+   :public-group-status                   "सार्वजनिक"
+   :active-online                         "अनलाइन"
+   :active-unknown                        "अज्ञात"
+   :available                             "उपलब्ध"
+   :no-messages                           "सन्देशहरु खाली"
+   :suggestions-requests                  "अनुरोधहरु"
+   :suggestions-commands                  "आदेशहरु"
+   :faucet-success                        "धारा अनुरोध प्राप्त भएको छ"
+   :faucet-error                          "धारा अनुरोध त्रुटि"
+
+   ;;sync
+   :sync-in-progress                      "सिङ्क भइरहेको छ॰॰॰"
```

Comment:
I'm going to go with मिलान for now

but "एकनास बनाइंदै छ" for "Syncing", likewise 
"एकनास छ" for "In Sync" also are equivalent isn't it? perhaps even more common words for conveying the meaning and purpose?